src/servers/ZoneServer2016/handlers/commands/commands.ts

```diff
@@ -3397,7 +3397,43 @@ export const commands: Array<Command> = [
       console.log(server.weatherManager.weather);
       server.weatherManager.sendUpdateToAll(server, client, false);
     }
-  }
+  },
+  {
+    name: "stats",
+    permissionLevel: PermissionLevels.DEFAULT,
+    keepCase: true,
+    execute: async (server: ZoneServer2016, client: Client) => {
 
+      const characterData = await server._db
+        .collection("characters")
+        .findOne({ characterId: client.character.characterId });
```

Comment:
specify the serverId even tho a characterId collision is extremely rare ( and never happened yet )